js/whitebit.js

```diff
@@ -974,64 +974,65 @@ module.exports = class whitebit extends Exchange {
          * @param {object} params extra parameters specific to the whitebit api endpoint
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
          */
-        let method = undefined;
         await this.loadMarkets ();
         const market = this.market (symbol);
         const request = {
             'market': market['id'],
             'side': side,
             'amount': this.amountToPrecision (symbol, amount),
         };
-        const stopPrice = this.safeNumber2 (params, 'stopPrice', 'activationPrice');
-        if (stopPrice !== undefined) {
-            // it's a stop order
+        const isLimitOrder = (type === 'limit');
+        const isMarketOrder = (type === 'market');
+        const stopPrice = this.safeNumber2 (params, 'triggerPrice', 'stopPrice');
```

Comment:
where does 'triggerPrice' come from? In the docs I'm only seeing `activation_price` 🤔 
![image](https://user-images.githubusercontent.com/43336371/181767161-a6c0bb4d-59e3-4077-bb6d-b37030e8591a.png)